### PR TITLE
New version for Android

### PR DESCRIPTION
**Pull request by Noxturnix**
Now user can hack easier than before.

### CODE REVIEW

File: Android/kahoot-hack.c [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
...gister the nicknames \"alex1\", \"alex2\", ..., \"alex100\".\n\n2. kahoot-rand - connect to a game an arbitrary number of times (e.g. 100) and answer each question randomly. If you connect with enough names, one of them is bound to win.\n\n3. kahoot-profane - circumvent Kahoot's profanity detector, allowing you to join with any nickname (but with extra length restrictions; it has to be short).\n\n4. kahoot-play - play kahoot regularly-as if you were using the online client.\n\n5. kahoot-html - I have notified Kahoot and they have fixed this issue. It used to allow you to join a game of kahoot a bunch of times with HTML-rich nicknames. This messes with the lobby of a kahoot game.\n\n6. kahoot-crash - trigger an exception on the host's computer. This no longer prevents the game from functioning, so it is a rather pointless \"hack\"\n\n7. kahoot-xss - since I discovered this security hole, I contacted Kahoot and they fixed it. This used to run arbitrary JavaScript code on the host's computer. This exploited a bug with the pre-game player list, which did not sanitize HTML tags. The exploit itself was rather complicated due to the fact that nicknames are limited to 15 characters.\n\n");
```
Comment (bjorn3) [inline Android/kahoot-hack.c]: Please put every output line on a seperate code line.

File: Android/kahoot-hack.c [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+do{
+printf("Which hack do you want to use? [1-7]: ");
```
Comment (bjorn3) [inline Android/kahoot-hack.c]: Please add indentation

**Comment by Noxturnix** [comment]
Done!

**Comment by unixpickle** [comment]
I notice that the C program is vulnerable to about a gazillion kinds of buffer overrun attack. Perhaps you should use use safer methods like `strncat`, `strncpy`. I don't know about a safe alternative to `scanf`. Suppose, for example, that the user enters a long value for number of bots to join (e.g. "1000000000000000000"). That could cause the C program to crash with a Segmentation fault, giving no meaningful error and causing someone to file a silly git issue on this repository.

**Comment by Noxturnix** [comment]
Oh sorry I forgot. This is not PHP. :p

I have no problem about the Segmentation fault. This is my result after I execute the program:
https://s4.postimg.org/63aidy5jh/res.png

File: Android/kahoot-hack.c [hunk 3]
```diff
@@ -0,0 +1,127 @@
+#include<stdio.h>
+#include<string.h>
+int main(){
+int hknm,pass;
+char spt[20],nob[50],pin[10],spc[2]=" ",cmd[200],nls[100],nme[20],cnl[1],cnly[2]="y",cnlY[2]="Y";
```
Comment (bjorn3) [inline Android/kahoot-hack.c]: Please make the variable names more descriptive.

**Comment by Zorphix** [comment]
@Noxturnix is there anyway to remove the bots from the game in termux? with a command perhaps?

File: Android/kahoot-hack.c [hunk 4]
```diff
@@ -0,0 +1,127 @@
+#include<stdio.h>
+#include<string.h>
+int main(){
+int Option,Loop;
```
Comment (bjorn3) [inline Android/kahoot-hack.c]: Please indent main body.

**Comment by Noxturnix** [comment]
I'm gonna add kahoot-auto to the old Android version.

**Pull request closed without merging**